### DISCUSSION TITLE
Welcome to Glyphs Discussions!

### DISCUSSION THREAD

**Discussion by gorango** [gorango/glyphs#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rjshxyz** [comment]
I just stumbled upon Glyphs and immediately felt why didn't I get to know about this before.
I wanted to let you guys know that you all are doing an amazing work.
Seriously this is fabulous and well thought out, CLI integration is a really great idea. 
Looking forward to contributing and spreading this amazing resource around.
